draft-ietf-taps-interface.md

```diff
@@ -320,6 +483,7 @@ following recommendations:
   because transport protocols implementing a given Property are not available on
   the platform.
 
+
```

Comment:
Random extra line? =)

Reply:
Oops, yea, that happened when I moved this part around on my branch...